cmd/tailscale/cli/funnel.go

```diff
@@ -20,7 +20,16 @@ import (
 	"tailscale.com/util/mak"
 )
 
-var funnelCmd = newFunnelCommand(&serveEnv{lc: &localClient})
+var funnelCmd = func() *ffcli.Command {
+	se := &serveEnv{lc: &localClient}
+	// This flag is used to switch to an in-development
+	// implementation of the tailscale funnel command.
+	// See https://github.com/tailscale/tailscale/issues/7844
+	if os.Getenv("TAILSCALE_FUNNEL_DEV") == "on" {
```

Comment:
@maisem sounds good, how should the CLI then register the new funnel codepath? A new command name? Like `tailscale funnel_dev 4545` instead of `tailscale funnel 4545`? Or shuold the env var happen on _both_ tailscaled and cli?

Reply:
either both, or a hidden new command

Reply:
@maisem, is the concern that someone finds and uses the environment variable to expose the work under development?

Reply:
> is the concern that someone finds and uses the environment variable to expose the work under development?
yes, or uses the LocalAPI directly without ever seeing the env var.

Experimental features that aren't fully baked yet or might have bugs should be disabled by default in tailscaled.